app/models/bet_option.rb

```diff
@@ -1,4 +1,7 @@
 class BetOption < ApplicationRecord
   has_many :users, through: :user_bets
   has_many :user_bets
+
+  scope :winners, -> { where(winner: true) }
+  scope :losers, -> { where(winner: [nil, false]) }
```

Comment:
Since this is a boolean field, lets make the db column default to false. This should usually be done for any boolean db column. That way your db column is guaranteed to be either true or false. ( also probably want to require this column and not allow it to be blank)